### PR TITLE
Fixes legacy k8s version support

### PR DESCRIPTION
**Pull request by foot**

### CODE REVIEW

File: pkg/plan/resource/rpm.go [hunk 1]
```diff
@@ -143,6 +143,8 @@ func (p *RPM) Apply(r plan.Runner, diff plan.Diff) (bool, error) {
 	if p.DisableExcludes != "" {
 		cmd = fmt.Sprintf("%s --disableexcludes %s", cmd, p.DisableExcludes)
 	}
+
+	cmd = fmt.Sprintf("%s --setopt=obsoletes=0", cmd)
```
Comment (bboreham) [inline pkg/plan/resource/rpm.go:147]: Bit simpler:
```suggestion
	cmd += " --setopt=obsoletes=0"
```

Comment (bboreham) [reply pkg/plan/resource/rpm.go:147]: Also I think I would comment this noting why it was added

Comment (yiannistri) [reply pkg/plan/resource/rpm.go:147]: Should this change affect all yum installs or should we isolate it to the kubelet install?

**Comment by bboreham** [comment]
We don't need this now that upstream Kubernetes has reissued packages, right?

**Comment by foot** [comment]
Indeed. This fix made it into the release-0.8 branch in f9c0a7bf69ae863a220eeeaaae30aab11949eb28 but I don't believe we need it on master anymore.

**Pull request closed without merging**